submissions/unabyband/popup-task/main.css

```diff
@@ -0,0 +1,302 @@
+html, body {
+    font-family: Arial, Helvetica, sans-serif;
+    width: 100%;
+    height: 100%;
+    margin: 0;
+    padding: 0;
+    border: 0;
+}
+
+.header {
+    display: flex;
+    justify-content: flex-end;
+    padding: 20px 0;
+    position: relative;
+}
+
+.top_menu {
+    display: flex;
+    padding: 0 15px;
+    align-items: center;
+}
+
+.top_menu__item {
+    line-height: 30px;
+    padding: 0 10px 0 10px;
+    font-size: 14px;
+    text-decoration: none;
+    color: rgba(0, 0, 0, 0.75);
+}
+
+.top_menu__item-container:hover {
+    text-decoration: underline;
+}
+
+.top_menu__item_bell {
+    margin-left: 5px;
+    width: 20px;
+    height: 25px;
+    display: block;
+    cursor: pointer;
+    background: url("../icons/bell.png") center center no-repeat;
+    background-size: 20px 25px;
+}
+
+.top_menu__item_user {
+    display: block;
+    height: 32px;
+    width: 32px;
+    border-radius: 50%;
+    background: url("../icons/favicon.png") center center no-repeat;
+    background-size: 32px 32px;
+}
+
+.top_menu__item_bell:hover, .top_menu__item_user:hover, .button_popup:hover {
+    opacity: 75%;
+}
+
+.input_popup {
+    position: absolute;
+    opacity: 0;
+}
+
+.input_popup:checked ~ .popup {
+    visibility: visible;
+}
+
+.popup {
+    visibility: hidden;
+    display: flex;
+    width: 320px;
+    height: 430px;
+    position: absolute;
+    box-sizing: border-box;
+    top: 62px;
+    right: 15px;
+    border: 1px solid #ccc;
+    box-shadow: 0 2px 10px rgba(0, 0, 0, 0.2);
+    background: #ffffff;
+    flex-direction: column;
```

Comment:
it is easier to read if for example flex properties will be placed one by one

Reply:
Yeah, I'm agree, thanks for your notice.